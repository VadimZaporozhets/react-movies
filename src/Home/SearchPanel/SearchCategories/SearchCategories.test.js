import React from 'react';
import { SearchCategoriesComponent } from './index';
import { shallow } from 'enzyme';

describe('<SearchCategoriesComponent />', () => {
    it('should render search categories component', () => {
        const props = {
            classes: expect.any(Object),
            searchBy: 'Title',
            handleSearchByChange: jest.fn()
        };

        const wrapper = shallow(<SearchCategoriesComponent {...props} />);

        expect(wrapper).toMatchSnapshot();
    });
});
