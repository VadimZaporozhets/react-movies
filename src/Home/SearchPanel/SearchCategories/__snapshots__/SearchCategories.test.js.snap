// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchCategoriesComponent /> should render search categories component 1`] = `
<WithStyles(Grid)
  container={true}
  spacing={40}
>
  <WithStyles(Grid)
    item={true}
    sm={12}
  >
    <WithStyles(Typography)
      gutterBottom={true}
      inline={true}
      variant="button"
    >
      Search by:
    </WithStyles(Typography)>
    <WithStyles(Button)
      color="primary"
      id="search-by-title"
      onClick={[MockFunction]}
      size="small"
      value="title"
      variant="outlined"
    >
      Title
    </WithStyles(Button)>
    <WithStyles(Button)
      color="primary"
      onClick={[MockFunction]}
      size="small"
      value="genres"
      variant="outlined"
    >
      Genre
    </WithStyles(Button)>
  </WithStyles(Grid)>
</WithStyles(Grid)>
`;
